mungegithub/github/github.go

```diff
@@ -2523,7 +2523,11 @@ func (config *Config) ForEachIssueDo(fn MungeFunction) error {
 	}
 
 	// It's a new day, let's restart from scratch.
-	if time.Now().Format("Jan 2 2006") != config.since.Format("Jan 2 2006") {
+	pst := time.FixedZone("PacificStandardTime", -8*60*60 /* seconds offset from UTC */)
+	// Use PST timezone to determine when its a new day.
+	nowPST := time.Now().In(pst)
+	sincePST := config.since.In(pst)
+	if nowPST.Format("Jan 2 2006") != sincePST.Format("Jan 2 2006") {
```

Comment:
I personally would not create the temporary variables `nowPST` and `sincePST` since they only save a few characters.

I think this is good enough:
```
if time.Now().In(pst).Format("Jan 2 2006") != config.since.In(pst).Format("Jan 2 2006") {
```